plugins/notion/server/mock-data.ts

```diff
@@ -0,0 +1,85 @@
+import { ProsemirrorDoc } from "@shared/types";
+import sample from "lodash/sample";
+
+const pageWithoutLinks: ProsemirrorDoc = {
```

Comment:
New fixture and mostly working converter are here if you want to pull commits across

https://github.com/outline/outline/compare/tom/notion-converter?expand=1